Source/Fuse.Controls.ScrollView/Scroller.uno

```diff
@@ -198,9 +200,26 @@ namespace Fuse.Gestures
 		{
 			get { return _scrollable == null ? GesturePriority.Lower : _scrollable.GesturePriority; }
 		}
+
+		bool _pressed;
+		void StartInvalidateVisual()
+		{
+			if (!_pressed)
+				UpdateManager.AddAction(_scrollable.InvalidateVisual);
+			_pressed = true;
+		}
+		void StopInvalidateVisual()
+		{
+			if (_pressed)
+				UpdateManager.RemoveAction(_scrollable.InvalidateVisual);
+			_pressed = false;
+		}
+
+		
 		
 		GestureRequest IGesture.OnPointerPressed(PointerPressedArgs args)
 		{
+			StartInvalidateVisual();
```

Comment:
This should go into the `OnCapture` function instead. There's no guarantee the scrollview gets capture when `OnPointerPressed` is called.